feature-group-definitions/css-has.yml

```diff
@@ -0,0 +1,2 @@
+spec: https://drafts.csswg.org/selectors-4/#relational
```

Comment:
I'd prefer `has.yml` (or perhaps `has-pseudo-class.yml`) for this file (and similar for the other `css-*.yml` files below). More context here in https://github.com/web-platform-dx/feature-set/issues/24#issuecomment-1363913244.

Reply:
I quite like the approach of using the shortest reasonable name and making them longer to disambiguate. So just "has" seems good to me here.